### PR TITLE
Three new token metadata+assets [HUNT, STUFF, DRIP]

### PR DESCRIPTION
**Pull request by rebelC0der**
# Hello MELD Team! &#128736;
I am submitting this pull request to add details for three Cardano tokens: Drip (DripDropz), Hunt (DexHunter), and STUFF (STUFF.io) in the repository. I have followed the prescribed guidelines and ensured that all necessary details are accurate and align with the required data structure. Below is an overview of the tokens and the associated changes.
- [x] I have checked that the format of my PR follows the contribution guidelines

### Token Overview &#128221;
- Token Names: Drip, Hunt, STUFF
- Token Tickers: DRIP, HUNT, STUFF
- Blockchain: Cardano (for all tokens)
- Token Type: Native Cardano tokens (for all tokens)
- Primary Use Cases: 
  - Drip: Utility token for the DripDropz platform, used for token distribution, rewards, and governance.
  - Hunt: Utility token for the DexHunter platform, used for incentivizing liquidity, governance, and staking rewards.
  - STUFF: Utility token for the STUFF.io platform (formerly Book.io), used for NFT book purchases, content distribution, and creator rewards.

### Organization Details &#127970;
- Organization Names: DripDropz, DexHunter, STUFF.io
- Websites: 
  - DripDropz: https://dripdropz.io
  - DexHunter: https://dexhunter.io
  - STUFF.io: https://stuff.io
- Mission & Vision:
  - DripDropz: To simplify and decentralize token distribution on Cardano, making it easy for projects to reach their communities.
  - DexHunter: To provide a comprehensive decentralized exchange (DEX) aggregator and analytics platform, helping users find the best trading opportunities on Cardano.
  - STUFF.io: To revolutionize digital content ownership by providing a decentralized platform for NFTs, including books, music, and other media, where creators and consumers directly interact.

#### For Additions:
- Community Demand: 
  - High demand for simplified, user-friendly token distribution services (DripDropz)
  - Growing interest in DEX aggregators and advanced trading tools on Cardano (DexHunter)
  - Increasing popularity of decentralized platforms for digital content ownership and NFT marketplaces (STUFF.io)
- Unique Value Proposition:
  - Drip: Easy and transparent token distribution with community governance features.
  - Hunt: Advanced trading tools and aggregation for better price discovery and liquidity optimization on Cardano.
  - STUFF: A decentralized content marketplace empowering creators and consumers through NFT technology.
- Technical Robustness: 
  - All tokens are built on Cardano's secure and scalable infrastructure.
  - DripDropz provides a decentralized, secure, and transparent token distribution model.
  - DexHunter leverages Cardano’s smart contract capabilities to aggregate and analyze DEX data.
  - STUFF.io uses blockchain technology to ensure authenticity, ownership, and immutability of digital content.
- Compliance and Security: 
  - All tokens adhere to Cardano's security standards and best practices.
  - Regular audits and updates are conducted to maintain platform integrity and user security.

### Rationale for Inclusion/Modification &#128200;
- Benefits to the Repository: 
  - Adds diverse Cardano tokens representing token distribution (Drip), DEX aggregation (Hunt), and NFT-based digital content ownership (STUFF).
  - Enhances the variety of use cases and services available in the MELD ecosystem.
- Impact on Community:
  - Provides exposure to innovative projects focusing on token distribution, DEX trading, and digital content ownership.
  - Attracts a broader range of users interested in decentralized finance, trading, and NFT-based content platforms on Cardano.

### CODE REVIEW

**Comment by rebelC0der** [comment]
These three are also solid Cardano projects, that are not on CmC yet, so no `SMSslug` included. I will add slugs as soon as these tokens are listed on CmC.

File: static/tokens/DRIP.svg
Comment (thuyvu17) [inline static/tokens/DRIP.svg:1]: it should be 24x24

File: networks/cardano/tokens/51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "",
+    "decimals": 6,
+    "mldId": "cardano-STUFF-51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b",
+    "orgName": "Stuff.io",
+    "orgUrl": "https://stuff.io",
+    "tokenAddress": "51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b",
+    "tokenColorTheme": 2,
+    "tokenIcon": "/static/tokens/STUFF.svg",
+    "tokenName": "BOOK",
```
Comment (thuyvu17) [inline networks/cardano/tokens/51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b.json:10]: wrong token name?

**Comment by thuyvu17** [comment]
please rebase your branch

File: networks/cardano/tokens/51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b.json (hunk 1 above)
Comment (rebelC0der) [reply networks/cardano/tokens/51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b.json:10]: This one is a tricky one. This project was changing from `Book,io` -> `Stuff.io`. `Asset Name` is still set to `Book`:

image

Do you think we should keep the `Book` or still swap for `Stuff`?

Comment (rebelC0der) [reply networks/cardano/tokens/51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b.json:10]: I am now checking with the `Stuff.io` team to see if they will update `BOOK` -> `STUFF`.

**Pull request merged**